ouroboros-consensus-shelley-test/src/Test/ThreadNet/Infra/Shelley.hs

```diff
@@ -529,3 +536,121 @@ mkKeyHashVrf = SL.hashVerKeyVRF . deriveVerKeyVRF
 
 networkId :: SL.Network
 networkId = SL.Testnet
+
+{-------------------------------------------------------------------------------
+  Temporary Workaround
+-------------------------------------------------------------------------------}
+
+eraIndTxBodyHash' ::
+     forall era body.
+     ( HashAnnotated body era
+     , HashIndex body ~ EraIndependentTxBody
+     )
+  => body
+  -> Shelley.Spec.Ledger.Keys.Hash
+       (Cardano.Ledger.Era.Crypto era)
+       EraIndependentTxBody
+eraIndTxBodyHash' = coerce . hashAnnotated
+
+mkAllegraSetDecentralizationParamTxs ::
```

Comment:
Comment here about this being a temporary bridge, soon to be replaced by the rewrite. In particular, we plan to replace this overall approach before Alonzo.